src/main/java/be/seeseemelk/mockbukkit/entity/LivingEntityMock.java

```diff
@@ -476,13 +475,6 @@ public void setRemoveWhenFarAway(boolean remove)
 		throw new UnimplementedOperationException();
 	}
 
-	@Override
-	public EntityEquipment getEquipment()
-	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
-	}
-
```

Comment:
Why did you move this to `SimpleMobMock` and `SimpleMonsterMock`?

Reply:
Because `EntityEquipment` is now mocked too, `#getEquipment()`  should not default to an `UnimplementedOperationException` in my opinion.
That's why I made the method abstract, so it can be decided on a per-class basis.
`SimpleMobMock` and `SimpleMonsterMock` currently do not implement the equipment, so they will now default to `UnimplementedOperationException` as before.